### ISSUE TITLE
Browser unexpectedly reopens after opening another browser with second selenideConfig in inNewBrowser lambda expression

### ISSUE THREAD

**Issue by ilya-corp** [selenide/selenide#2859]
## The problem
Opening of browsers with another SelenideConfig in the lambdas (method inNewBrowser) should not affect work of the main browser.

## Details
In the attached example opening of browser through inNewBrowser brings to restarting of main browser and clearing attached cookie. This is wrong.
The expected bahaviour is the following:
- browser in the lambda should be opened with own config
- executing of the open method for main browser should only open page without restarting browser. Cookie should not disappear.

There are two clones of original test from Yaroslav:
```
// This test will fail
@Test()
public void selenide751failed1() {
  final SelenideConfig original = new SelenideConfig();
  Selenide.open("https://selenide.org/", original);
  WebDriverRunner.getWebDriver().manage().addCookie(new Cookie("bober", "kurwa"));
  Selenide.webdriver().shouldHave(cookie("bober", "kurwa"));
  inNewBrowser(() -> {
    SelenideConfig target = new SelenideConfig();
    Selenide.open("https://github.com", target);
    Selenide.webdriver().shouldNotHave(cookie("bober", "kurwa"));
  });
  Selenide.open("https://selenide.org/blogs", original);
  Selenide.webdriver().shouldHave(cookie("bober", "kurwa"));
}

```

```
// This test is currently passed
@Test()
  public void selenide751failed2() {
    final SelenideConfig original = new SelenideConfig();
    Selenide.open("https://selenide.org/", original);
    WebDriverRunner.getWebDriver().manage().addCookie(new Cookie("bober", "kurwa"));
    Selenide.webdriver().shouldHave(cookie("bober", "kurwa"));
    Selenide.open("https://selenide.org/blogs", original);
    Selenide.webdriver().shouldHave(cookie("bober", "kurwa"));
  }
```

## Tell us about your environment
* **Selenide Version**: 7.5.1
* **Chrome\Firefox\IE Version**:  Chrome 128
* **Browser Driver Version**:
* **Selenide-Appium Version**:
* **IOS\Android\WEB version**:
* **Selenium Version**: 
* **OS Version**: Windows 10 Pro

## Code To Reproduce Issue [ Good To Have ]
```
// The original test from Ilya
@Test(groups = {"draft"})
    public void selenide751failed() {
        final SelenideConfig selenideConfigOriginal = new SelenideConfig();       
        Selenide.open("https://selenide.org/", selenideConfigOriginal);
        WebDriverRunner.getWebDriver().manage().addCookie(new Cookie("bober", "kurwa"));
        Assert.assertEquals(Objects.requireNonNull(WebDriverRunner.getWebDriver().manage().getCookieNamed("bober")).getValue(), "kurwa");
        inNewBrowser(() -> {
            SelenideConfig secondConfig = new SelenideConfig();
            Selenide.open("https://github.com", secondConfig);
        });
        Selenide.open("https://google.ru/", selenideConfigOriginal);
        Assert.assertEquals(Objects.requireNonNull(WebDriverRunner.getWebDriver().manage().getCookieNamed("bober")).getValue(), "kurwa");
    }

```

**Comment by ilya-corp** [comment]
:(

**Comment by asolntsev** [comment]
@ilya-corp The fix has been released in Selenide 7.6.0

**Comment by ilya-corp** [comment]
Thank you, Andrey!
